fhirG5/CodeSystem-shr-actor-LanguageQualifierCS.html

```diff
@@ -1,168 +1,172 @@
-
-Downloads: <a href="CodeSystem-shr-actor-LanguageQualifierCS.xml" download>XML</a>, <a href="CodeSystem-shr-actor-LanguageQualifierCS.json" download>JSON</a>, <a href="CodeSystem-shr-actor-LanguageQualifierCS.ttl" download>Turtle</a>
-
-<p> </p>
-
-<h3>References</h3>
-
-This code system is used by the following value sets:
-
-<ul>
- <li><a href="ValueSet-shr-actor-LanguageQualifierVS.html">SHR LanguageQualifierVS ValueSet</a></li>
-</ul>
-
-
-<!-- ==============END CONTENT END CONTENT=================== -->
-
-        </div>
-      </div>  <!-- /inner-wrapper -->
-    </div>  <!-- /row -->
-  </div>  <!-- /container -->
-</div>  <!-- /segment-content -->
-
-        </div>
-      </div>  <!-- /inner-wrapper -->
-    </div>  <!-- /row -->
-  </div>  <!-- /container -->
-</div>  <!-- /segment-content -->
-
-<div id="segment-footer" class="segment">  <!-- segment-footer -->
-  <div class="container">  <!-- container -->
-   <div class="inner-wrapper">
-    <p>Standard Health Record Collaborative &copy; FHIR STU3 (IG Publisher v3.0.0-11681) generated
-        on Mon, Jul 31, 2017 12:50-0400.</p>
-   </div>  <!-- /inner-wrapper -->
-  </div>  <!-- /container -->
-</div>  <!-- /segment-footer -->
-
-<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
-  <div class="container">  <!-- container -->
-  </div>  <!-- /container -->
-</div>  <!-- /segment-post-footer -->
-
-</body>
-</html>
+---
+redirect_from:
+  - /fhir/CodeSystem/shr-actor-LanguageQualifierCS/
+---
```

Comment:
What is this redirect stuff at the top of the file?  That's not in the previous version.